o11y/honeycomb/honeycomb.go

```diff
@@ -26,6 +26,21 @@ type honeycomb struct {
 	metricsProvider o11y.ClosableMetricsProvider
 }
 
+func (h *honeycomb) StartGoldenTrace(ctx context.Context, name string) context.Context {
+	//TODO implement me
+	panic("implement me")
```

Comment:
is there any risk in trying to use this in a service that isn't OTel-enabled? I _think_ that you've already onboarded all the services, but I don't know if that's definitely true. If not, would it make more sense to just leave these as TODOs with a no-op return of the provided context?

Reply:
mebe - but if anyone is using these right now - i would want them to panic! - kinda even the ot1l ones - they are not ready. If anyone else uses them now - they have no idea what will happen :p

Reply:
but yah - totes safer to remove the panic - good call :)